docs/reST/ref/mask.rst

```diff
@@ -45,8 +45,31 @@ Starting from pygame 1.9.5 masks with width or height 0 are supported.
 
 .. class:: Mask
 
-   | :sl:`pygame object for representing 2d bitmasks`
-   | :sg:`Mask((width, height)) -> Mask`
+   | :sl:`pygame object for representing 2D bitmasks`
+   | :sg:`Mask(size=(width, height)[, fill=False]) -> Mask`
```

Comment:
Maybe this should show two signatures instead? I.e.

```
| :sg:`Mask(size=(width, height)) -> Mask`
| :sg:`Mask(size=(width, height), fill=False) -> Mask`
```

Reply:
Yes it should. I've made the requested change.